source/components/handling-events.md

```diff
@@ -0,0 +1,130 @@
+Instead of having to register event listeners on elements you'd like to
+respond to, simply implement the name of the event you want to respond to
+as a method on your component.
+
+For example, imagine we have a template like this:
+
+```hbs
+{{#x-clickable}}
+  This is a clickable area!
+{{/x-clickable}}
+```
+
+Let's implement `x-clickable` such that when it is
+clicked, an alert is displayed:
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    alert("XClickableComponent was clicked!");
+  }
+});
+```
+
+Browser events may bubble up from the target component to each parent component
+in succession, until the root component. To enable bubbling `return true;` from
+the event handler method in your component.
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    Ember.Logger.info("XClickableComponent was clicked!");
+    return true;
+  }
+});
+```
+
+### Event Names
+
+  All of the event handling approaches described above respond to the same set
+  of events. The names of the built-in events are listed below. (The hash of
+  built-in events exists in `Ember.EventDispatcher`.) Additional, custom events
+  can be registered by using `Ember.Application.customEvents`.
+
+  Touch events:
+
+  * `touchStart`
+  * `touchMove`
+  * `touchEnd`
+  * `touchCancel`
+
+  Keyboard events
+
+  * `keyDown`
+  * `keyUp`
+  * `keyPress`
+
+  Mouse events
+
+  * `mouseDown`
+  * `mouseUp`
+  * `contextMenu`
+  * `click`
+  * `doubleClick`
+  * `mouseMove`
+  * `focusIn`
+  * `focusOut`
+  * `mouseEnter`
+  * `mouseLeave`
+
+  Form events:
+
+  * `submit`
+  * `change`
+  * `focusIn`
+  * `focusOut`
+  * `input`
+
+  HTML5 drag and drop events:
+
+  * `dragStart`
+  * `drag`
+  * `dragEnter`
+  * `dragLeave`
+  * `dragOver`
+  * `dragEnd`
+  * `drop`
+
+###Sending Actions
+
+To have the click event from `x-clickable` affect the state of your application,
+simply send an action to the component's template context (the controller):
```

Comment:
I would also prefer if this paragraph was phrased around converting events to actions, the current description is a bit jargony.